src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
@kd0aij Sorry for the delay, long weekend over here and I managed to get away.  I get similar performance from my simulation. 
http://logs.uaventure.com/view/RSXUbQ8V6toGXk3Jg3wdHY

While I can't speak to how the performance is or is not like your rascal the simulator is running correctly.  If you wanted the sim to be more representative of the flight dynamics of your aircraft you would have to change the values of the JSBSim xml file.  These are your aero coefficients and if you look in the file you will see that they are very basic.  High fidelity simulation would require running aero analysis tools like AVL on your aircraft configuration and generating a new JSBSim xml.